### PR TITLE
Major version updates

### PR DESCRIPTION
**Pull request by ConnorChristensen**
Hello everyone!

After months of work in the gaps between my time working on tickets, my self-assigned task of documenting stories on the ALF project draws to a close. This little wrapper helper project started as a way to make my stories look cleaner, and now that it has served me to document an entire Funda website project (albeit a small one), I have applied all the changes I needed to document a real Funda project.

I already know this code works well enough, the impacts of this not working perfectly is negligible (as this is a tool that never touches customer facing code), and I made a lot of changes here, so do not feel like you need to review this line-by-line.

I would, however, like your feedback regarding the API and the general way this project works. Hopefully, this pull request will affect you all, as I want to start integrating this into the Funda UI library, future projects, and get other teams to start integrating this on existing projects. You should be able to get a full picture of this project and how it works by just looking at the markdown files (readme and roadmap).

I have not polished this to perfection, and the tests are good enough, but not comprehensive, but as always, questions and suggestions for improvement are most welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,7 +138,75 @@ export const ShowFileNames = uploadButtonStory({
...
 ```
 
-## Automatic features
+### A full example
+
+This is an example of a component that makes use of every option in the `story`
+function configuration.
+
+```js
+import { story, book } from '@funda/storybook-config';
```
Comment (dine) [inline README.md:147]: from '@funda/storybook-wrapper-helper'

**Review by dine — APPROVED** [review]
Small thing on the import, but overall looks good! 🚀

**Review by bartderijk — APPROVED** [review]
Nice!

File: src/types/book.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    // storybook will think that we would like the decorator to be applied both
+    // at the book and story level. If we pass in an array of strings, for our
+    // decorators, we can have a more simple interface for our developers, while
+    // also being able to determine which decorators are passed in at the story
+    // level versus the book level.
+    decorators?: Array<string>;
+    title?: string;
+    // since we have a fallback layout, no need to make the layout optional
+    parameters?: BookParameters;
+    argTypes?: AnyObject;
```
Comment (Zishrodrigues) [inline src/types/book.ts:36]: Any reason to not type `argTypes`? Looking at https://storybook.js.org/docs/vue/api/argtypes it seems like the object has set types.

**Review by Zishrodrigues — APPROVED** [review]
Looks good! I'd reconsider if you need the `anyObject` type where you've used it. Other than that I find all the types files really clear to read.

File: README.md (hunk 1 above)
Comment (ConnorChristensen) [reply README.md]: Thank you!

File: src/types/book.ts (hunk 2 above)
Comment (ConnorChristensen) [reply src/types/book.ts]: Good point. I updated this to be more helpful.

**Comment by ConnorChristensen** [comment]
I pushed some changes that addresses feedback from reviewers, as well as adds the feature to be able to integrate figma designs in stories.

**Comment by ConnorChristensen** [comment]
Rebased the branch to squish some fixup commits

**Pull request merged**